ethereum/api/src/test/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/parameters/FilterParameterTest.java

```diff
@@ -224,6 +238,83 @@ public void emptyListAsSubTopicDecodesCorrectly() throws JsonProcessingException
             emptyList());
   }
 
+  @Test
+  public void emptyListOfTopicsInJsonGivesInEmptyListInFilterParameter()
```

Comment:
Method naming in this test class is the format _[premise] should [outcome]_. 

I'd suggest a name such as `emptyTopicsListShouldDeserializeAsEmptyList()`